contrib/TestHarness2/test_harness/run.py

```diff
@@ -367,6 +367,13 @@ def run(self):
             command += ['-b', 'on']
         if config.crash_on_error:
             command.append('--crash')
+        if self.long_running:
```

Comment:
I think you could use `config.long_running` here if you didn't want to plumb it through as an argument

Reply:
yeah, that's true. It seems all the copying from _config_ here is unnecessary.